examples/user_interfaces/toolmanager.py

```diff
@@ -83,10 +114,21 @@ def set_lines_visibility(self, state):
 fig.canvas.manager.toolbar.add_tool('zoom', 'foo')
 
 # Remove the forward button
-fig.canvas.manager.toolmanager.remove_tool('forward')
+tool_mgr.remove_tool('forward')
 
 # To add a custom tool to the toolbar at specific location inside
 # the navigation group
 fig.canvas.manager.toolbar.add_tool('Hide', 'navigation', 1)
 
+for i, c in enumerate(['yellowgreen', 'forestgreen']):
+    sidebar = fig.canvas.manager._get_toolbar()
```

Comment:
I don't think we should use this "internal" method here. In that case, I would suggest to create a method in the base class that "creates" a toolbar